compiler/dyno/tools/chpldoc/chpldoc.cpp

```diff
@@ -969,6 +1051,12 @@ struct RstResultBuilder {
       os_ << "\n";
     }
 
+    showDeprecationMessage(node, indentComment);
+    return commentShown;
+  }
+
+  template<typename T>
```

Comment:
Does this need to be templated? Why not just make `node` a `const Decl*` instead?